src/name_bazaar/ui/pages/manage_names_page.cljs

```diff
@@ -182,7 +182,8 @@
 
             full-subname (str subname "." full-name)
             correct-subname? (and (not (empty? subname))
```

Comment:
So empty subname is in general valid, except this one case?

Reply:
Yeah, because the function is used in the UI for form validation. In that case empty string is valid.
However this line checks, whether the `Register` button will be enabled or not, and clearly empty subname is not a valid in this case.